homeassistant/helpers/config_validation.py

```diff
@@ -682,6 +703,14 @@ def uuid4_hex(value: Any) -> str:
     return result.hex
 
 
+def fake_uuid4_hex(value: Any) -> str:
+    """Validate a fake v4 UUID generated by random_uuid_hex."""
+    regex = re.compile(r"^[0-9a-f]{32}$")
```

Comment:
We shouldn't compile this on every invocation, as this validator is called often.

Although Python does cache the last X compilations, we can't guarantee that's us. Better store it as a constant.

Reply:
Fixed in 6e72682